### PR TITLE
Pay Test VPC Peering

### PR DESCRIPTION
**Pull request by rorymalcolm**
What
----

Added VPC peering details for Pay's test environment

How to review
-------------

Sense check with @danworth or @rjbaker 

Who can review
--------------

PaaS team members

### CODE REVIEW

**Comment by rjbaker** [comment]
I've checked the details in the destination account/VPC and they're correct.

File: terraform/prod.vpc_peering.json [hunk 1]
```diff
@@ -70,5 +70,11 @@
         "account_id": "234617505259",
         "vpc_id": "vpc-08b7d799f1691fd11",
         "subnet_cidr": "172.16.36.0/22"
+    },
+    {
+        "peer_name": "govuk-pay-test",
+        "account_id": "734876687119",
+        "vpc_id": "vpc-031070d650a39111e",
+        "subnet_cidr": "172.17.36.0/22"
```
Comment (tlwr) [inline terraform/prod.vpc_peering.json:78]:
```suggestion
        "subnet_cidr": "172.16.40.0/22"
```
Can this be continuous within the 172.16 range (ignore govuk-infrastructure-test to see the pattern)

Comment (rorymalcolm) [reply terraform/prod.vpc_peering.json]: Updated - maybe sense check with @rjbaker but I think this is to spec now

File: terraform/prod.vpc_peering.json [hunk 2]
```diff
@@ -70,5 +70,11 @@
         "account_id": "234617505259",
         "vpc_id": "vpc-08b7d799f1691fd11",
         "subnet_cidr": "172.16.36.0/22"
+    },
+    {
+        "peer_name": "govuk-pay-test",
+        "account_id": "734876687119",
+        "vpc_id": "vpc-0170bee868cc6fb34",
+        "subnet_cidr": "172.17.40.0/22"
```
Comment (rjbaker) [inline terraform/prod.vpc_peering.json:78]: Should this not be 172.16.40.0/22

Comment (rorymalcolm) [reply terraform/prod.vpc_peering.json:78]: it should, thanks

**Review by paroxp — APPROVED** [review]

**Pull request merged**